app/src/main/res/values/themes.xml

```diff
@@ -35,6 +35,7 @@
     <item name="colorAccent">@color/component_color_shared_dialogs_primary_color</item>
     <item name="colorBackgroundFloating">@color/component_color_shared_item_background_solid_color</item>
     <item name="android:textColorPrimary">@color/component_color_shared_primary_dark_text_color</item>
+    <item name="textColorAlertDialogListItem">@color/component_color_shared_primary_dark_text_color</item>
```

Comment:
"dark_text_color" makes it seem like it's only for night mode--what does this color actually mean?

Also, this will affect all alert dialogs, correct? It may be worth checking all the others to make sure they haven't changed. If they have, you should add before/after screenshots for each in the PR description to show that the changes are expected & correct.

Reply:
Thanks @BenHenning ,

1. "dark_text_color" denotes the dark color(dark black) we have by default ( that is in normal mode not in dark mode), in dark-mode we have White color. That's why @rt4914 Introduced this color.

2. Yes this effect all the dialogs whose contains list of items, which uses OppiaDialogFragmentTheme.